whisper.cpp

```diff
@@ -540,37 +540,29 @@ struct whisper_decoder {
     bool has_ts;    // have we already sampled a non-beg timestamp token for the current segment?
 
     // new token probs, logits and logprobs after the last whisper_decode (1-dimensional array: [n_vocab])
-    std::vector<float> probs;
-    std::vector<float> logits;
-    std::vector<float> logprobs;
+    std::vector<float> probs{};
```

Comment:
Fought with some read access denied exception for almost 2h, only to find out that it's a bug in compiler (only for debug): https://github.com/microsoft/STL/issues/1934

The workaround is just to initialize the struct (otherwise, it's pretty hard to debug)